packages/suite-data/files/translations/en.json

```diff
@@ -630,9 +640,9 @@
 	"TR_EXCHANGE_PROVIDER": "Provider",
 	"TR_EXCHANGE_RATE": "Price",
 	"TR_EXCHANGE_RECEIVE_ACCOUNT_QUESTION_TOOLTIP": "This is the account within your wallet where you'll find your coins once the transaction is finished.",
-	"TR_EXCHANGE_RECEIVE_NON_SUITE_ACCOUNT_QUESTION_TOOLTIP": "Receiving account is outside of Suite.",
 	"TR_EXCHANGE_RECEIVE_ADDRESS_QUESTION_TOOLTIP": "This is the specific alphanumeric address that will receive your coins. Verify this address on your Trezor.",
-	"TR_EXCHANGE_RECEIVE_NON_SUITE_ADDRESS_QUESTION_TOOLTIP": "This is the specific alphanumeric address that will receive your coins.",
+	"TR_EXCHANGE_RECEIVE_NON_SUITE_ACCOUNT_QUESTION_TOOLTIP": "TR_EXCHANGE_RECEIVE_NON_SUITE_ACCOUNT_QUESTION_TOOLTIP",
+	"TR_EXCHANGE_RECEIVE_NON_SUITE_ADDRESS_QUESTION_TOOLTIP": "TR_EXCHANGE_RECEIVE_NON_SUITE_ADDRESS_QUESTION_TOOLTIP",
```

Comment:
@LukasRada These were added by you [here](https://github.com/trezor/trezor-suite/commit/5cf184412289711766c25ed43c3f6f8fce448968). Can we publish this as it is or shall we rather wait for (quick) fix?

Reply:
Please wait for fix. To which branch I'll create PR? feat/crowdin-sync or develop?

Reply:
Can you maybe commit (and push) directly to this branch? I'll then reorder the commits and run the sync again.

Reply:
Done.

Reply:
Thanks. I force pushed the commits now in reverse order and with your new translations in the localization file.

Resolving.,